### DISCUSSION TITLE
How to link a horse's facing to its passenger

### DISCUSSION THREAD

**Discussion by ZakShearman** [Minestom/Minestom#1541] [Help]
Hi there, the video demonstrates my issue

RED = horse pos
GREEN = passenger pos

pos is printed by overriding the `tick` method of the horse (printing happens after super#tick is called).

https://user-images.githubusercontent.com/34372536/204141293-bc2fc517-3bbf-47ba-ac24-3d2158b2ae83.mp4

The horse and passenger pos are identical (except for the y obviously), yet the passenger faces a different direction to that of the horse.

**Comment by GamerDuck123** [comment]
Are you using Pos, Vec, or Point to set their locations. Try using Pos#withView(yaw, pitch) you can read more on the differences here: https://wiki.minestom.net/world/coordinates

**Reply by ZakShearman** [reply]
I'm simply adjusting the existing Pos. I'm not sure how it would be possible to use Vec or Point to change the yaw as they don't contain those properties.
